server/methods/core/orders.app-test.js

```diff
@@ -0,0 +1,122 @@
+import { Meteor } from "meteor/meteor";
+import { Factory } from "meteor/dburles:factory";
+import { Reaction } from "/server/api";
+import { Orders, Products, Notifications } from "/lib/collections";
+import { expect } from "meteor/practicalmeteor:chai";
+import { sinon } from "meteor/practicalmeteor:sinon";
+import Fixtures from "/server/imports/fixtures";
+
+Fixtures();
+
+describe("orders test", () => {
+  let methods;
+  let sandbox;
+  let order;
+
+  before(() => {
+    Factory.create("user");
+    methods = {
+      cancelOrder: Meteor.server.method_handlers["orders/cancelOrder"]
+    };
+  });
+
+  beforeEach((done) => {
+    sandbox = sinon.sandbox.create();
+    // });
+    sandbox.stub(Orders._hookAspects.insert.before[0], "aspect");
+    sandbox.stub(Orders._hookAspects.update.before[0], "aspect");
+    sandbox.stub(Meteor.server.method_handlers, "inventory/register", function () {
+      check(arguments, [Match.Any]);
+    });
+    sandbox.stub(Meteor.server.method_handlers, "orders/sendNotification", function () {
+      check(arguments, [Match.Any]);
+    });
+    sandbox.stub(Meteor.server.method_handlers, "inventory/sold", function () {
+      check(arguments, [Match.Any]);
+    });
+
+    Factory.create("shop");
+    order = Factory.create("order");
+    return done();
+  });
+
+  afterEach(() => {
+    Orders.remove({});
+    sandbox.restore();
+  });
+
+  function spyOnMethod(method, id) {
+    return sandbox.stub(Meteor.server.method_handlers, `orders/${method}`, function () {
+      check(arguments, [Match.Any]); // to prevent audit_arguments from complaining
+      this.userId = id;
+      return methods[method].apply(this, arguments);
+    });
+  }
+
+  describe("orders/cancelOrder", () => {
+    it("should return an error if user is not admin", (done) => {
```

Comment:
You don't want to use arrow functions within Mocha tests. https://mochajs.org/#arrow-functions